src/api/api.js

```diff
@@ -0,0 +1,159 @@
+import { useEffect, useState } from 'react'
+
+const URL = (idInstance, apiTokenInstance, action) =>
```

Comment:
КАПСОМ только константы называют, если это функция, то есть лучше дать семантичное имя
```suggestion
const createApiUrl = (idInstance, apiTokenInstance, action) =>
```